### PR TITLE
chore: iOS 14.2 + 13.4 validation

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: test/functional/basic/find-e2e-specs.js [hunk 1]
```diff
@@ -332,6 +332,10 @@ describe('XCUITestDriver - find', function () {
       await B.delay(TEST_PAUSE_DURATION);
     });
     it('should find visible elements', async function () {
+      // This test is failing on 14.2... skipping it for now
```
Comment (KazuCocoa) [inline test/functional/basic/find-e2e-specs.js:335]: mm.
I met a similar issue in ruby_lib_core. iPhone 12 simulator x iOS 14.2 returned the wrong element coordinate but iPhone 11 x iOS 14.2 one worked well. Potentially such combination stuff could be here...

File: test/functional/basic/gesture-e2e-specs.js [hunk 2]
```diff
@@ -159,11 +159,13 @@ describe('XCUITestDriver - gestures', function () {
       await driver.execute('mobile: scroll', {direction: 'down'});
       await driver.elementByAccessibilityId('Steppers').click();
 
+      await B.delay(1000);
```
Comment (KazuCocoa) [inline test/functional/basic/gesture-e2e-specs.js:162]: Can be a `retryInterval` for `await driver.elementByAccessibilityId('Increment');`?
(L168 as well)

**Review by KazuCocoa — APPROVED** [review]

File: test/functional/basic/gesture-e2e-specs.js (hunk 2 above)
Comment (dpgraham) [reply test/functional/basic/gesture-e2e-specs.js:162]: I tried with that and the test failed.

File: test/functional/basic/find-e2e-specs.js (hunk 1 above)
Comment (dpgraham) [reply test/functional/basic/find-e2e-specs.js:335]: I couldn't reproduce locally. I don't think I have time to be testing different combinations in Azure.

**Pull request merged**